egs/babel_multilang/s5/run-4-anydecode-langs.sh

```diff
@@ -0,0 +1,451 @@
+fi
+
+#####################################################################
+#
+# Audio data directory preparation
+#
+#####################################################################
+echo ---------------------------------------------------------------------
+echo "Preparing ${dataset_kind} data files in ${dataset_dir} on" `date`
+echo ---------------------------------------------------------------------
+if [ ! -f  $dataset_dir/.done ] ; then
+  if [ "$dataset_kind" == "supervised" ]; then
+    if [ "$dataset_segments" == "seg" ]; then
+      . ./local/datasets/supervised_seg.sh || exit 1
+    elif [ "$dataset_segments" == "uem" ]; then
+      . ./local/datasets/supervised_uem.sh || exit 1
+    elif [ "$dataset_segments" == "pem" ]; then
+      . ./local/datasets/supervised_pem.sh || exit 1
+    else
+      echo "Unknown type of the dataset: \"$dataset_segments\"!";
+      echo "Valid dataset types are: seg, uem, pem";
+      exit 1
+    fi
+  elif [ "$dataset_kind" == "unsupervised" ] ; then
+    if [ "$dataset_segments" == "seg" ] ; then
+      . ./local/datasets/unsupervised_seg.sh
+    elif [ "$dataset_segments" == "uem" ] ; then
+      . ./local/datasets/unsupervised_uem.sh
+    elif [ "$dataset_segments" == "pem" ] ; then
+      ##This combination does not really makes sense,
+      ##Because the PEM is that we get the segmentation
+      ##and because of the format of the segment files
+      ##the transcript as well
+      echo "ERROR: $dataset_segments combined with $dataset_type"
+      echo "does not really make any sense!"
+      exit 1
+      #. ./local/datasets/unsupervised_pem.sh
+    else
+      echo "Unknown type of the dataset: \"$dataset_segments\"!";
+      echo "Valid dataset types are: seg, uem, pem";
+      exit 1
+    fi
+  else
+    echo "Unknown kind of the dataset: \"$dataset_kind\"!";
+    echo "Valid dataset kinds are: supervised, unsupervised, shadow";
+    exit 1
+  fi
+
+  if [ ! -f ${dataset_dir}/.plp.done ]; then
+    echo ---------------------------------------------------------------------
+    echo "Preparing ${dataset_kind} parametrization files in ${dataset_dir} on" `date`
+    echo ---------------------------------------------------------------------
+    make_plp ${dataset_dir} exp/$lang/make_plp/${dataset_id} plp/$lang
+    touch ${dataset_dir}/.plp.done
+  fi
+
+
+  if [ ! -f ${dataset_dir}_hires/.mfcc.done ]; then
+    dataset=$(basename $dataset_dir)
+    echo ---------------------------------------------------------------------
+    echo "Preparing ${dataset_kind} MFCC features in  ${dataset_dir}_hires and corresponding iVectors in exp/$lang/nnet3/ivectors_${dataset}${ivector_suffix} on" `date`
+    echo ---------------------------------------------------------------------
+    if [ ! -d ${dataset_dir}_hires ]; then
+      utils/copy_data_dir.sh $data/$dataset $data/${dataset}_hires
+    fi
+
+    mfccdir=mfcc_hires/$lang
+    steps/make_mfcc.sh --nj $my_nj --mfcc-config conf/mfcc_hires.conf \
```

Comment:
this should be make_mfcc or make_mfcc_pitch_online depending on the use_pitch variable